### DISCUSSION TITLE
Environment specific Pack Configuration

### DISCUSSION THREAD

**Discussion by johnbonda** [StackStorm/st2#5544] [Q&A]
Hi, is it possible to have Environment specific pack configuration in Stackstorm? We have stackstorm deployed in different environments (DEV & PROD), and would like to have some configuration that is specific to the environment.

**Comment by arm4b** [comment]
You can have different st2 key/value configuration set between the environments and rely in the st2 pack configs on these K/V settings.

**Reply by johnbonda** [reply]
I tried a pack configuration like this:
config.yaml:
```
dev:
    foo: bar
prod:
    foo: baz
```
and set `st2 key set env dev`
I tried to access it in my action yaml with `{{ config_context[st2kv.system.env].foo }}` but it gives me this error:
```
ERROR: 400 Client Error: Bad Request
MESSAGE: Failed to render parameter "foo": dict object has no element <st2common.services.keyvalues.KeyValueLookup object at 0x7f59f044d978> for url: http://st2api:9101/executions
```
